B/bliss/build_tarballs.jl

```diff
@@ -41,7 +40,8 @@ platforms = supported_platforms(;experimental=true)
 
 # The products that we will ensure are always built
 products = [
-    LibraryProduct("libbliss", :libbliss)
+    LibraryProduct("libbliss", :libbliss),
+    ExecutableProduct("bliss", :bliss),
```

Comment:
@matbesancon I also added the executable as a product here, I figured it could be useful (also to ensure the file is named correctly)